Data/Samples/PacMan/RuntimeConfigs/Plugins.ddl

```diff
@@ -23,8 +23,3 @@ Plugin
 	string %Path{"ezParticlePlugin"}
 	bool %LoadCopy{false}
 }
-Plugin
-{
-	string %Path{"ezXBoxControllerPlugin"}
-	bool %LoadCopy{false}
-}
```

Comment:
This obviously removes a feature from the Windows version, which isn't great. Either we keep it and just ignore that on Linux you'll see a plugin load error message, or you make the plugin compile on Linux, even if it doesn't do anything, so that it's not a problem to keep this reference.
OR you make that plugin actually work on Linux, but no idea how complicated that is.

Reply:
This is not a intentional change. The editor simply removes this entry from the `Plugins.ddl` on Linux, because this plugin does not exist on Linux. Building a ezXBoxControllerPlugin doesn't make sense for Linux, as glfw already coveres all gamepad & joystick input. We need a concept of platform specific plugins.

Reply:
Yeah, something like that. But I guess not in this PR. I would revert this change, though.